CONTRIBUTING.md

```diff
@@ -15,30 +15,33 @@
 
 We expect contributors to abide by our underlying
 [code of conduct](https://dev.to/code-of-conduct). All conversations and
-discussions on GitHub (issues, pull requests) and across dev.to must be
-respectful and harassment-free.
+discussions on GitHub (issues, pull requests) and across
+[https://dev.to](dev.to) must be respectful and harassment-free.
 
 Remember that communication is the lifeblood of any Open Source project. We are
 all working on this together, and we are all benefiting from this software. It's
 very easy to misunderstand one another over asynchronous, text-based
-conversations: When in doubt, assume everyone you're interacting within this project has the best intentions.
+conversations: When in doubt, assume everyone you're interacting within this
+project has the best intentions.
 
 If you feel another member of the community has violated our Code of Conduct,
 you may anonymously contact the team with our
 [abuse report form](https://dev.to/report-abuse).
 
 ### Where to contribute
 
-All [issues](https://github.com/thepracticaldev/dev.to/issues) labeled with
-`ready for dev` are up for grabs.
+All [issues](https://github.com/thepracticaldev/dev.to/issues) labeled
+[ready for dev](https://github.com/thepracticaldev/dev.to/issues?q=is%3Aissue+is%3Aopen+label%3A%22ready+for+dev%22)
+and
+[type: bug](https://github.com/thepracticaldev/dev.to/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+bug%22)
+all are up for grabs.
```

Comment:
we should encourage people to let them know that both `ready for dev` and `type: bug` are *ALWAYS* up for grubs, bugs don't require approval